### ISSUE TITLE
Video Game Sales Analysis

### ISSUE THREAD

**Issue by krishna16sharma** [OpenCodeEra/ML-X#7]
## What ?

I would like to add a notebook for data exploration and analysis of [Video Games Sales Dataset](https://www.kaggle.com/datasets/gregorut/videogamesales)

- ### Dataset

## Code of Conduct

- [x] I agree to follow this project's Code of Conduct

<br>
<br>

> **Kindly assign it to me under _HacktoberFest'23_** 
> **I will surely complete it ASAP**

**Comment by AKACHI-4** [comment]
Hey, @krishna16sharma, yeah, go ahead with the task. Don't forget to take notes while writing the notebook so that others can understand it as well.

**Comment by ZestyZeeshan** [comment]
hello i want to contribute in this cool project video game sales dataset, if  u allow me i can use different notebook whick contains all your efforts and some additional of mine.
plz assign me

**Comment by AKACHI-4** [comment]
> hello i want to contribute in this cool project video game sales dataset, if u allow me i can use different notebook whick contains all your efforts and some additional of mine. plz assign me

Hey @ZestyZeeshan, yeah... kindly mention the dataset you will use, and then go ahead.

**Comment by AKACHI-4** [comment]
@ZestyZeeshan, kindly remember that while you are writing the notebook, add some notes and link your dataset so others can understand it as well.

As there is already a merged PR, so when you commit your notebook, give it the filename `notebook_1.ipynb`.

**Comment by ZestyZeeshan** [comment]
hi, @AKACHI-4 thanks for response i am using https://www.kaggle.com/datasets/thedevastator/global-video-game-sales-ratings/code dataset .
sure i will use notes as well so that its easy for user users as well
[archive.zip](https://github.com/OpenCodeEra/ML-X/files/12783816/archive.zip)

**Comment by AKACHI-4** [comment]
Yeah, I think you can go with the old notebook to make some changes on it... no need to add a new one, but make sure that you are not altering the latest changes. Just add your notes and scripts. Hope you understand.

**Comment by ZestyZeeshan** [comment]
sorry for disturbing you again n again, from the above i had to use your notebook and add notes and my own script in that notebook itself , i dont need to make another notebook.

**Comment by ZestyZeeshan** [comment]
https://www.kaggle.com/datasets/gregorut/videogamesales dataset

**Comment by AKACHI-4** [comment]
Yah, you have to add this link there in notebook ..

**Comment by ZestyZeeshan** [comment]
ok sure.

**Comment by ZestyZeeshan** [comment]
plz check my pr
